ajax/libs/buy-button-js/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "buy-button-js",
+  "filename": "buybutton.js",
```

Comment:
`filename` points to the main file of the lib which in this case is `buybutton.min.js`